### PR TITLE
translation of !info moves & !info items & !script commands

### PR DESCRIPTION
**Pull request by ReuschelCGN**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/lib/poracleMessage/commands/script.js [hunk 1]
```diff
@@ -22,12 +22,12 @@ exports.run = async (client, msg, args, options) => {
 		const translator = client.translatorFactory.Translator(language)
 
 		if (!await util.commandAllowed(commandName)) {
-			await msg.react('🚫')
+			await msg.react('ðŸš«')
```
Comment (jfberry) [inline src/lib/poracleMessage/commands/script.js:25]: Was this intentional? Seems broken

File: src/lib/poracleMessage/commands/script.js [hunk 2]
```diff
@@ -241,7 +241,6 @@ exports.run = async (client, msg, args, options) => {
 					message += `${prefix}gym ${teamNames[gym.team]}`
 
 					if (gym.slot_changes) message += ' slot_changes'
-					if (gym.battle_changes) message += ' battle_changes'
```
Comment (jfberry) [inline src/lib/poracleMessage/commands/script.js:244]: Did this get lost?

File: src/lib/poracleMessage/commands/script.js (hunk 1 above)
Comment (ReuschelCGN) [reply src/lib/poracleMessage/commands/script.js]: Was fixed before you write!

File: src/lib/poracleMessage/commands/script.js (hunk 2 above)
Comment (ReuschelCGN) [reply src/lib/poracleMessage/commands/script.js]: Yes your right, start translation last wednesday and it was changed after that... reinsert it, sorry

File: locale/de.json [hunk 3]
```diff
@@ -300,6 +302,7 @@
     "Instinct":"Intuition",
     "invasion":"Invasion",
     "italian":"Italienisch",
+    "items":"Gegenstand",
```
Comment (Fabio1988) [inline locale/de.json:305]: Items is plural. Isn't it?

**Pull request merged**